builder/builder.go

```diff
@@ -22,6 +22,8 @@ type dockerAuthData struct {
 	Password   string `yaml:"password"`
 	Repository string `yaml:"repository"`
 	Version    string `yaml:"version"`
+	AssignmentEnvImage   string `yaml:"assignmentEnvImage"`
```

Comment:
why do we need image and version information for authentication?

Reply:
This was the initial code. This shall eventually change after implementing code for installation scripts.